projects/ngx-qgrid-plugins/src/lib/pane/pane.component.ts

```diff
@@ -35,37 +39,43 @@ export class PaneComponent implements OnInit {
 			observeReply(model[`${state}Changed`])
 				.subscribe((e: GridEventArg<any>) => {
 					if (!prop || e.hasChanges(prop)) {
-						this.close('right');
-						this.open('right');
+						this.updateAndOpen('right')
 					}
 				});
 		}
 	}
 
-	open(side: 'right') {
+	open(side: PaneSide, value: any = null) {
 		const { table, model } = this.plugin;
 
-		let value = null;
 		const scope = this.parse();
-		if (scope) {
+		if (scope && !value) {
```

Comment:
should check `isUndefined(value)` because if the value is `0` or `false` that won't work